app/messages/farmer-apply.js

```diff
@@ -0,0 +1,18 @@
+import { v4 as uuidv4 } from 'uuid'
+import { NotifyClient } from 'notifications-node-client'
+
+const farmerApply = async (message) => {
```

Comment:
Could do with making this name more generic as it could be used for different templates.

Reply:
Also agree, have changed the name to `sendNotification`.